protocol/xmrmaker/claim.go

```diff
@@ -147,19 +162,55 @@ func (s *swapState) discoverRelayersAndClaim() (ethcommon.Hash, error) {
 			continue
 		}
 
-		return resp.TxHash, nil
+		log.Infof("DHT relayer's claim validated block=%d gas-used=%d tx=%s",
```

Comment:
```suggestion
		log.Infof("relayer's claim included in block=%d gas-used=%d tx=%s",
```

Reply:
We have a separate log depending on whether the Maker got the receipt from a DHT relayer or from the Taker. I wanted the logged message to be unique, so the log reader can tell which type was used. In this case the word "validated" does not mean "mined" or "included in block", it means that we did additional validation in `waitForClaimReceipt` which checks the receipt status and the logs. Do you have a suggestion to improve the message with that additional context in mind?

Reply:
okay, i'd say `included in block and validated` then? because when i read the initial log, it wasn't clear that the transaction had actually been included in the chain, it just sounds like something was validated

Reply:
New version is:
```
log.Infof("DHT relayer's claim included and validated %s", common.ReceiptInfo(receipt))
```